pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineGRPCServerMultiStageOOMAccountingIntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.integration.tests;
+
+import org.testng.annotations.BeforeTest;
+
+
+public class OfflineGRPCServerMultiStageOOMAccountingIntegrationTest
+    extends OfflineGRPCServerOOMAccountingIntegrationTest {
+  @BeforeTest
+  void enableMultiStage() {
+    setUseMultiStageQueryEngine(true);
+  }
```

Comment:
I don't get this. IICU we cannot run MSE using GRPC server

Reply:
There is a similar test https://github.com/apache/pinot/blob/master/pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineGRPCServerMultiStageIntegrationTest.java 

Am I testing the wrong GRPC end points ?

Reply:
I've removed the test.